pelican-bootstrap3/templates/includes/comments.html

```diff
@@ -8,7 +8,7 @@
             /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
             var disqus_shortname = '{{ DISQUS_SITENAME }}'; // required: replace example with your forum shortname
 
-            {% if article %}
+            /*{% if article %}
```

Comment:
Okay, great. Would you please remove the commented-out code and amend the relevant commit(s)?